### ISSUE TITLE
debug automatically restart

### ISSUE THREAD

**Issue by tsiens** [r0x0r/pywebview#1548]
Can PyWebview automatically restart the service like Flask's debug mode when Python code is changed? Currently, it seems that PyWebview's debug feature only opens the F12 console.
Any guidance on achieving automatic restarts or alternative solutions would be greatly appreciated.
Thank you!

**Comment by r0x0r** [comment]
No, it cannot. I haven't researched this feature, nor I am sure if it is possible.

**Comment by tsiens** [comment]
> No, it cannot. I haven't researched this feature, nor I am sure if it is possible.

I think I really need this feature. Currently, every time there is a code change, I need to close the exe and run it again。
If it can be achieved, thank you very much

**Comment by tsiens** [comment]
> py-mon or nodemon may be an answer to your problems

Thank you very much. Although it is a cold update and cannot be updated as hot as Flask, but at least it is more convenient than manually updating it now
